docs/content/configuration/creating_volumegroupclass.md

```diff
@@ -0,0 +1,32 @@
+# Creating a VolumeGroupClass
+
+Create a VolumeGroupClass YAML file to enable volume groups.
+
+Volume groups allows users to create PersistentVolumeClaim (PVC) groups. PVCs can be dynamically managed through the volume groups created. This allows actions across all PVCs within the same volume group at onces.
```

Comment:
This allows actions across all PVCs within the same volume group at once*

Reply:
updated